.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature Request
+about: Suggest a/an feature/enhancement to the Jenkins X docs project
+labels:
+- area/docs
+- documentation
```

Comment:
Are these labels appropriate or should we include `kind/feature` or any other label?

Reply:
`area/docs` and `documentation` labels in a docs repo dont add much information.
I prefer `kind/feature` (or `kind/enhancement`) and `kind/bug`.

Reply:
you mean `kind/bug` for both bug report and feature request?